### PR TITLE
feat(wallet-connection): Add getAdminBootstrap functionality to wallet connection

### PR DESCRIPTION
**Pull request by samsiegart**

### CODE REVIEW

File: packages/wallet-connection/test/src/mock-agoric-iframe-messenger.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  class MockAgoricIframeMessenger extends LitElement {
+    static get properties() {
+      return { src: { type: String } };
+    }
+
+    constructor() {
+      super();
+      this.src = '';
+      this.sentData = null;
+      window[mockController].ref = this;
```
Comment (samsiegart) [inline packages/wallet-connection/test/src/mock-agoric-iframe-messenger.js]: I tried passing a regular object reference as the controller but it seemed that whenever I modified it it tried to reinstantiate the class and caused an error.

File: packages/wallet-connection/src/agoric-wallet-connection.js [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+import { interpret } from 'robot3';
+
+import { makeConnectionMachine } from './states.js';
+import { makeAgoricIframeMessenger } from './agoric-iframe-messenger.js';
+
+// TODO: Use something on agoric.app instead.
+const DEFAULT_LOCATOR_URL =
+  'https://local.agoric.com/?append=/wallet-bridge.html';
+
+export const makeAgoricWalletConnection = (
```
Comment (samsiegart) [inline packages/wallet-connection/src/agoric-wallet-connection.js]: Sorry about the diff. The only thing I changed in this file was this top part.

File: packages/wallet-connection/test/src/mock-agoric-iframe-messenger.js (hunk 1 above)
Comment (michaelfig) [reply packages/wallet-connection/test/src/mock-agoric-iframe-messenger.js]: I didn't specify very clearly, but I expected that the `makeXXX` functions would only be explicitly called once at the top level of the test, and that the API exported by the different files would not change, only be added to (yes, I know that. the lint rules complain about this, but that can be turned off).

I'll rearrange the code in that way.

File: packages/wallet-connection/src/agoric-wallet-connection.js (hunk 2 above)
Comment (michaelfig) [reply packages/wallet-connection/src/agoric-wallet-connection.js]: If the diff happens in the same file, that's fine... I just turn off the whitespace change detection:

image

File: packages/wallet-connection/test/agoric-wallet-connection.test.js [hunk 3]
```diff
@@ -1,11 +1,73 @@
...
+);
+
+// Expose the captp innards for the wallet connection.
+customElements.define(
+  'agoric-wallet-connection',
+  makeAgoricWalletConnection(makeMockCapTP),
+);
 
 describe('AgoricWalletConnection', () => {
+  // eslint-disable-next-line no-unused-vars
```
Comment (samsiegart) [inline packages/wallet-connection/test/agoric-wallet-connection.test.js]: I think we can remove this, I was just using it for debugging

**Comment by samsiegart** [comment]
LGTM

**Review by michaelfig — APPROVED** [review]
Great job!  LGTM.

**Pull request merged**